.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    # Release when a tag is pushed (should work with GitHub "release" button also)
+    tags:
+      - 'v/[0-9]+(\.[0-9]+)*(-alpha\.[0-9]+)?/'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

Comment:
Shall we use 3.10?

Reply:
Not too sure, I've always done latest stable release -  1 minor version, but we could bump it to 10

Reply:
On reflection, I think it's all g and doesn't actually matter for `release.yml` which we use as long as that version is also unit tested ay

Reply:
[b7e2df3](https://github.com/datavaluepeople/phoenix-scrape/pull/2/commits/b7e2df3e71c728f8dd9c28fbe779bdaffed11a06)